src/App.tsx

```diff
@@ -1,109 +1,64 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+  const totalItems = items.length;
+
+  const from = (currentPage - 1) * Number(perPage) + 1;
+  const to =
+    currentPage * Number(perPage) > 42 ? 42 : currentPage * Number(perPage);
```

Comment:
You're converting `perPage` to a number each time you use it, which is unnecessary since the state is already set to a number. You should remove the `Number(perPage)` conversion to use the state value directly.